#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_openvas_nvt-sync-var.dpatch by Javier Fernandez-Sanguino <jfs@debian.org>
##
## DP: plugins should be in /var

@DPATCH@
diff -urNad openvas-server-2.0.2~/openvas-nvt-sync.in openvas-server-2.0.2/openvas-nvt-sync.in
--- openvas-server-2.0.2~/openvas-nvt-sync.in	2009-07-27 21:05:58.000000000 +0000
+++ openvas-server-2.0.2/openvas-nvt-sync.in	2009-07-27 21:06:12.000000000 +0000
@@ -35,7 +35,7 @@
 
 # these locations should be correct if standard ./configure had
 # been applied.
-NVT_DIR="$libdir/openvas/plugins"
+NVT_DIR="$localstatedir/openvas/plugins"
 
 # The URL of the plugin feed
 FEED=rsync://rsync.openvas.org:/nvt-feed
